src/typeorm/migrations/1654520916132-ward.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class ward1654520916132 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'wards',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+            length: '255',
+          },
+          {
+            name: 'districts_id',
+            type: 'int',
+          },
+          {
+            name: 'created_at',
+            type: 'timestamp',
+            isNullable: true,
+            default: 'now()',
+          },
+          {
+            name: 'updated_at',
+            type: 'timestamp',
+            isNullable: true,
+            default: 'now()',
+          },
+        ],
+      }),
+      true,
+    );
+    await queryRunner.createForeignKey(
+      'wards',
+      new TableForeignKey({
+        columnNames: ['districts_id'],
```

Comment:
```suggestion
        columnNames: ['district_id'],
```

FK ko phải là số nhiều